docs/components/box/Example.vue

```diff
@@ -3,24 +3,24 @@
 </script>
 
 <template>
-  <h3>Neutral</h3>
-  <div class="component">
-    <w-box neutral>
-      <p>This is the <strong>neutral</strong> variant of the box element</p>
-    </w-box>
-  </div>
-
-  <h3>Info</h3>
-  <div class="component">
-    <w-box info>
-      <p>This is the <strong>info</strong> variant of the box element</p>
-    </w-box>
-  </div>
-
-  <h3>Bordered</h3>
-  <div class="component">
-    <w-box bordered>
-      <p>This is the <strong>bordered</strong> variant of the box element</p>
-    </w-box>
+  <div class="component space-y-16">
+    <div>
+      <h3 class="h4">Neutral</h3>
+      <w-box neutral>
+        <p>This is the <strong>neutral</strong> variant of the box element</p>
```

Comment:
Why strong here and "" in elements doc? :)

Reply:
Ah. I didn't think about aligning these, but I guess that it would be nice to align the syntax examples to what you actually see? Even though the working examples are often much too polluted with other stuff, so then you would probably want a more clean syntax example in those cases? Do you agree? 🙂